packages/runtime/test/src/runtime.ts

```diff
@@ -1441,13 +1442,56 @@ describe("Helper functions", function () {
 		});
 	});
 });
-describe("Funtions related to runtime chain", function () {
+
+describe("Atomic Transaction Composer",function () {
 	let runtime: Runtime;
+	const notABI = "not-ABI.json";
+	const notJSON = "not-JSON.teal";
+	const networkRuntimeNotDefined = "network-runtime-undefined.json";
+	const correctABI = "correct-ABI.json";
+	useFixture("atomic-transaction-composer")
 
 	this.beforeEach(function () {
 		runtime = new Runtime([]);
 	});
 
+	it("Should throw an error if file is not found", function () {
+		assert.throws(() => {
```

Comment:
use expectRuntimeError for checking the exact thrown error.

Reply:
this error is not thrown by us. Its from external package

Reply:
ok